aries-site/src/pages/components/tabs.mdx

```diff
@@ -42,6 +43,25 @@ To maintain a clean, consistent presentation, all content should be aligned to t
 - Apply `justify="start"` to left-align the tabs and have their bottom border extend the full-width of the parent container.
 - Content within the Tab should be flush with the left edge of the Tabs component and have a vertical padding of `medium` to provide spacing and room for the content to breathe.
 
+### Responsive Tabs
+
+When horizontal space is limited, only the tabs that fit will be displayed. The remaining tabs can be accessed by using the previous and next arrows or with the keyboard.
+
+### Responsive Behavior
+
+- Previous and next arrows will be present if the horizontal tabs are fully outside the viewport.
+- Previous and next arrows are disabled if the horizontal tab has no more tabs to view towards the left or right.
+- Clicking on the previous and next arrows will show additional tabs. On a small screen the arrows will show 1 additional tab. On medium or large screens, clicking the arrows will show 3 additional tabs.
+
+<Example
+  docs="https://v2.grommet.io/tabs?theme=hpe#props"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tabs/TabResponsiveExample.js"
```

Comment:
In addition to `code`, can we add the `github` prop and link to the example's file source?

Reply:
Done in latest commit